projects/1.20-fabric/assets/0-modrinth-mod/more_food/lang/zh_cn.json

```
@@ -0,0 +1,178 @@
+{
+  "item.food_mod.sausage_roll": "香肠卷",
+  "item.more_food.gummy_bear": "小熊软糖",
+  "item.more_food.fairy_floss": "棉花糖",
+  "item.more_food.sweet_berry_donut": "甜浆果甜甜圈",
+  "item.more_food.chocolate_iceblock": "巧克力冰棒",
+  "item.food_mod.fish_and_chips": "炸鱼薯条",
+  "item.more_food.glow_berry_icecream": "发光浆果冰淇淋",
+  "item.food_mod.crushed_cooked_cocoa_beans": "熟可可豆碎",
+  "item.more_food.apple_pie": "苹果派",
+  "item.more_food.cocoa_butter": "可可脂",
+  "item.food_mod.raw_bacon": "生培根",
+  "item.more_food.plain_icecream": "原味冰淇淋",
+  "item.food_mod.bacon_and_eggs": "培根加鸡蛋",
+  "item.food_mod.golden_apple_iceblock": "金苹果冰棒",
+  "item.food_mod.sushi_roll": "寿司卷",
+  "item.food_mod.gummy_worm": "软糖虫",
+  "item.more_food.chocolate_donut": "巧克力甜甜圈",
+  "item.more_food.fried_rice": "炒饭",
+  "item.food_mod.sweet_berry_paste": "甜浆果酱",
+  "item.more_food.glow_berry_iceblock": "发光浆果冰棒",
+  "item.food_mod.cooked_rice": "熟米饭",
+  "item.food_mod.golden_apple_icecream": "金苹果冰淇淋",
+  "item.food_mod.glow_berry_iceblock": "发光浆果冰棒",
+  "item.food_mod.taco": "墨西哥卷饼皮",
+  "item.more_food.toffee_apple": "太妃苹果糖",
```

```suggestion
  "item.more_food.toffee_apple": "太妃苹果",
```
这种应该是一整个苹果泡到太妃糖浆里面整出来的吧

---

大概就是直接苹果和太妃糖直接合成（